packages/runtime-core/__tests__/rendererFragment.spec.ts

```diff
@@ -351,4 +355,121 @@ describe('renderer: fragment', () => {
     render(renderFn(['two', 'one']), root)
     expect(serializeInner(root)).toBe(`text<div>two</div>text<div>one</div>`)
   })
+
+  // #9200
+  test('stable fragment in unstable slot', () => {
```

Comment:
this test case is too complex to understand.
here is a [simple one](https://play.vuejs.org/#__DEV__eNp9U8lu2zAQ/ZUxe3ALRNYhN0EOuiCH9tAWbW9lDoo0splQJEFSjgFD/54ZKlYkZNFBIOe9Gb7ZTuKLc5tDj6IQZai9chECxt6BrsxuK0UMUlxJozpnfYQTeGxhgNbbDtbktp6gb7ZzT/ZNzheOSrA0tTWBou7tA2zZ/2P0PX6SpszHByk8XSJ2TlcR6QZQ3vYxWgOfa63qe5LB3tsV/0lOvSdtWOYjaXRIzxeJNrKTbKBvigwfatSa4BPsVYMwTJQ56ZCpljgrpswIRFllGdS269BEyLI50qjD1W3ly5wPU8h8kdILQ5mqRMeFXVxQyalirdpt7oI11JcTs6Wgp53S6H+5qKiiUhSQEMYqre3Dj2Tj4l6c7fUe6/tX7HfhyDYpfnsM6A+U6YTFyu8wjvD13594pPMEdrbpNbHfAf9gsLpnjSPta28akj3jJbXf09gos/sXro8RTTgnxUKZOSS+FDRGXKi3Un+We7m5TH7SDFTF8wjyYC/r/tykMmgbwVQdUsd5OKSAghvPA7AYolcmJOmcD0hr7Tu956ee+j4KWHDOu7fcOjym1WqwrXpN28fuzlsXCvi/Zn3rG0p2sUlieATtkj3p)